rca/events/forms.py

```diff
@@ -11,3 +11,7 @@ def __init__(self, *args, **kwargs):
         self.fields["contact_model_image"].label = "Image"
         self.fields["contact_model_text"].label = "Text"
         self.fields["contact_model_link_text"].label = "Link text"
+
+
+class EventIndexPageAdminForm(EventPageAdminForm):
+    pass
```

Comment:
Will this be added to later?

Reply:
Nope - it's pretty much the same as `EventPageAdminForm` for now, so rather than declare the same thing I've inherited it

Reply:
In that case could it be renamed something like `EventAdminForm` and used on both pages, kinda feels like a misuse of inheritance again.